tests/test_providers.py

```diff
@@ -711,7 +708,11 @@ def test_is_base_available(
     base = Base(name=name, channel=channel, architectures=architectures)
     valid, reason = providers.is_base_available(base)
 
-    assert (valid, reason) == (expected_valid, expected_reason)
+    assert valid == expected_valid
+    if reason is None:
```

Comment:
This implies that we probably have two tests here at this point: One for valid bases and one for invalid bases, with a reason check.

At that point it may also be worth changing the reason check into a regex check. For example, the modified test case above could be the following regex if you use `fullmatch`:

```
base 'ubuntu' channel '10.04' is not yet supported \(must be '.+'\)
```

(Try to keep the regexes simple as with the above example — `.+` and `.*` are generally fine to use since they signal to the reader tha we don't care about the specific contents of that particular substring.)

Reply:
OK, I will use regex to check substring.